components/clp-py-utils/clp_py_utils/serialization_utils.py

```diff
@@ -0,0 +1,23 @@
+import pathlib
+
+from strenum import StrEnum
+
+
+def serialize_enum(enum_value: StrEnum) -> str:
+    """
+    Serializes a StrEnum to its underlying value.
+
+    :param enum_value: A StrEnum instance.
```

Comment:
```suggestion
    :param enum_value:
```
or if you agree with the proposed renaming
```suggestion
    :param member:
```